contracts/rbn-staking/Redeemer.sol

```diff
@@ -100,6 +101,27 @@ contract Redeemer is Ownable {
     emit RBNRedeemed(amountToRedeem);
   }
 
+  /**
+   * @dev Redeems the rbn using manual amount
+   * @param _amount is the amount
+   */
+  function redeemRBN(uint256 _amount) external onlyOwner {
```

Comment:
You can refactor the two implementations of `redeemRBN` into a single function so we DRY

```solidity

function redeemRBN() {
  _redeemRBN(seizerImplementation.amountToRedeem(address(votingEscrowContract)));
}

function redeemRBN(uint amount) {
  _redeemRBN(amount);
}

function _redeemRBN(uint amount) {
}
```